src/index.ts

```diff
@@ -54,12 +56,14 @@ export const submitCancellableForm = async (
                 if (isTerminalState(newStatus)) {
                     statusHandler(newStatus, {
                         ...formData,
+                        submittedAt,
```

Comment:
We can assume submittedAt is already included in formData because it should have been already submitted before this line is triggered.

Reply:
so no need to merge submittedAt